skll/config.py

```diff
@@ -99,7 +99,7 @@ def __init__(self):
                                    'min_feature_count': 'Tuning',
                                    'models': 'Output',
                                    'num_cv_folds': 'Input',
-                                   'objective': 'Tuning',
```

Comment:
No, we do not leave it in. If the configuration file contains `objective`, objective will be converted into objectives and then it will start to validate the configuration file.